assets/src/scripts/monitoring-location/components/hydrograph/parameters.test.js

```diff
@@ -78,4 +104,43 @@ describe('monitoring-location/components/hydrograph/parameters module', () => {
             loadMedian: false
         });
     });
+
+    it('Expects clicking on a row will expand and contact the correct rows', function() {
+        store = configureStore(TEST_STATE);
+        drawSelectionList(div, store, '11112222');
+
+        const firstTargetRow = div.select('#container-row-00010');
+        const secondTargetRow = div.select('#container-row-72019');
+        expect(select('#expansion-container-row-00010').attr('hidden')).toBe('true');
+        expect(select('#expansion-container-row-72019').attr('hidden')).toBe('true');
```

Comment:
This is a minor style thing. Putting a line feed between each set of tests makes it easier to read. So a line feed after this line (and so on) would be good.

Reply:
Yup, does look better.